RA-16-Favorites-Icon

DenisGubernatorov opened the pull request:
1) Настройка кнопки добавления в избранное

DenisGubernatorov committed:
RA-16-Favorites-Icon
1) Настройка кнопки добавления в избранное

aGroupoid commented on app/src/main/java/com/example/recipesapp/RecipeFragment.kt:
```diff
@@ -108,6 +109,22 @@ class RecipeFragment : Fragment() {
+        setFavoritesButtonImage(R.drawable.ic_heart_40_empty)
+
+        binding.favoritesImage.setOnClickListener {
+            setFavoritesButtonImage(R.drawable.ic_heart_40)
```
Кнопка должна работать по принципу переключателя, повторное нажатие должно возвращать его в исходное положение

DenisGubernatorov replied:
Подумал что это функционал лучше было реализовать в следующем этапе. Когда будем рисовать актуально состояние и добавлять/удалять из избранного

DenisGubernatorov committed:
RA-16-Favorites-Icon
1) Доработка логики нажатия кнопки Избранное. Если было в избранном, то снимает выделение и наоборот

aGroupoid commented on app/src/main/java/com/example/recipesapp/RecipeFragment.kt:
```diff
@@ -108,6 +110,29 @@ class RecipeFragment : Fragment() {
+                null
+            }
+
+        binding.favoritesImage.setImageDrawable(drawable)
```
Лучше использовать сразу setImageResource, и передавать id картинки напрямую

aGroupoid reviewed (approved):
Молодец, по заданию верно! Поправь коммент по оформлению в следующем ПР
